pkg/dev_compiler/tool/input_sdk/patch/core_patch.dart

```diff
@@ -602,7 +602,10 @@ class Uri {
 @patch
 class _Uri {
   @patch
-  static bool get _isWindows => false;
+  static bool get _isWindows => _isWindowsCached;
+
+  static final bool _isWindowsCached =
+      JS('bool', 'process !== undefined && process.platform == "win32"');
```

Comment:
I have to check whether "win32" is sufficient (since I'm generally running 64-bit Windows).